src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyle.java

```diff
@@ -4,21 +4,20 @@
 ////////////////////////////////////////////////////////////////////////////////
 package com.puppycrawl.tools.checkstyle.checks.arraytypestyle;
 
-/**
- * Test case for ArrayTypeStyle (Java vs C)
- * @author lkuehne
- **/
+/* Config:
+ * javaStyle = true
```

Comment:
You can just write `Config: default`, if no properties were specified

Reply:
@timurt done